build/bin/network-validator.go

```diff
@@ -115,6 +113,7 @@ func ValidateReachability(host string, port int, tlsDisabled bool) error {
 		Timeout: *timeout,
 	}
 
+	// #nosec G402 -- Low chance of MITM, as the instance is short-lived, see OHSS-11465
```

Comment:
Did you mean to add this in this PR?

Reply:
yeah, I think i need to add that to skip the gosec as Jason commented on the card:
`[Linh Nguyen](https://issues.redhat.com/secure/ViewProfile.jspa?name=linnguye.openshift), I agree with your justification about disabling that check. My preference for disabling any checks is that the specific lines or blocks have the specific gosec check disabled — with the justification inline or cited — rather than disabling the check completely in the gosec options.`

Reply:
Can we please separate these into two different PRs?